static/app/views/explore/tables/fieldRenderer.tsx

```diff
@@ -38,10 +42,63 @@ interface FieldProps {
 }
 
 export function FieldRenderer({data, meta, unit, column}: FieldProps) {
-  const location = useLocation();
-  const organization = useOrganization();
   const userQuery = useExploreQuery();
   const setUserQuery = useSetExploreQuery();
+
+  return (
+    <BaseExploreFieldRenderer
+      data={data}
+      meta={meta}
+      unit={unit}
+      column={column}
+      userQuery={userQuery}
+      setUserQuery={setUserQuery}
+    />
+  );
+}
+
+export interface MultiQueryFieldProps extends FieldProps {
+  index: number;
+}
+
+export function MultiQueryFieldRenderer({
+  data,
+  meta,
+  unit,
+  column,
+  index,
+}: MultiQueryFieldProps) {
+  const queries = useReadQueriesFromLocation();
+  const userQuery = queries[index]!.query;
```

Comment:
Would it be safer to do something like this instead ```queries[index]?.query ?? ''```? If there's a possibility that `queries[index]` is undefined somehow